src/api/java/com/minecolonies/api/util/BlockPosUtil.java

```diff
@@ -715,6 +715,18 @@ public static Direction getXZFacing(final int pos1X, final int pos1Z, final int
         return Direction.getNearest(pos1X - pos2X, 0, pos1Z - pos2Z);
     }
 
+    /**
+     * Calculate in which direction a pos is facing. Ignoring y.
+     *
+     * @param pos      the pos.
```

Comment:
broken jdoc and,I feel like this method doesnt belong here

Reply:
git mees up, its a copy of the function above^^